lua/harpoon/tmux.lua

```diff
@@ -115,10 +118,15 @@ end
 function M.gotoTerminal(idx)
     log.trace("tmux: gotoTerminal(): Window:", idx)
     local window_handle = find_terminal(idx)
+    local specifier = "select-window";
+
+    if window_handle.pane then
+        specifier = "select-pane"
+    end
 
     local _, ret, stderr = utils.get_os_command_output({
         "tmux",
-        "select-window",
+        specifier,
```

Comment:
just FYI this could be done inline perhaps 
```lua
window_handle.pane and "select-pane" or "select-window"
```

Reply:
Much nicer, thank you @disrupted.